From fa96172dd1111d06da55cb36b2f93e5c82b57658 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Tue, 27 Nov 2012 11:05:42 +0800
Subject: [PATCH 104/304] ENGR00234893 mx5/6: refine the features in
 permission list

i.MX5 SMD:
Remove the Barometer feature from permission list,
add back the accelerometer and compass features.

i.MX6 SabreSD:
Remove the Gyroscope feature from permission list,
add back the accelerometer and compass features.

Incorrect permission feature list causes failure of
CTS case:
android.app.cts.SystemFeaturesTesttestSensorFeatures

Change-Id: I8de17398fb423e9f2fa7ae1c51ed829660f42d40
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx5x/imx53_smd.mk                   | 3 ++-
 imx6/sabresd/SabreSDProductCommon.mk | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/imx5x/imx53_smd.mk b/imx5x/imx53_smd.mk
index 00ae463..c039243 100644
--- a/imx5x/imx53_smd.mk
+++ b/imx5x/imx53_smd.mk
@@ -23,7 +23,8 @@ PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.hardware.wifi.xml:system/etc/permissions/android.hardware.wifi.xml \
     frameworks/native/data/etc/android.hardware.wifi.direct.xml:system/etc/permissions/android.hardware.wifi.direct.xml \
     frameworks/native/data/etc/android.hardware.sensor.light.xml:system/etc/permissions/android.hardware.sensor.light.xml \
-    frameworks/native/data/etc/android.hardware.sensor.barometer.xml:system/etc/permissions/android.hardware.sensor.barometer.xml \
+    frameworks/native/data/etc/android.hardware.sensor.accelerometer.xml:system/etc/permissions/android.hardware.sensor.accelerometer.xml \
+    frameworks/native/data/etc/android.hardware.sensor.compass.xml:system/etc/permissions/android.hardware.sensor.compass.xml \
     frameworks/native/data/etc/android.hardware.camera.front.xml:system/etc/permissions/android.hardware.camera.front.xml \
     frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
     frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml \
diff --git a/imx6/sabresd/SabreSDProductCommon.mk b/imx6/sabresd/SabreSDProductCommon.mk
index 805f07c..d62f3fa 100644
--- a/imx6/sabresd/SabreSDProductCommon.mk
+++ b/imx6/sabresd/SabreSDProductCommon.mk
@@ -18,7 +18,8 @@ PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.wifi.xml:system/etc/permissions/android.hardware.wifi.xml \
 	frameworks/native/data/etc/android.hardware.wifi.direct.xml:system/etc/permissions/android.hardware.wifi.direct.xml \
 	frameworks/native/data/etc/android.hardware.sensor.light.xml:system/etc/permissions/android.hardware.sensor.light.xml \
-	frameworks/native/data/etc/android.hardware.sensor.gyroscope.xml:system/etc/permissions/android.hardware.sensor.gyroscope.xml \
+	frameworks/native/data/etc/android.hardware.sensor.accelerometer.xml:system/etc/permissions/android.hardware.sensor.accelerometer.xml \
+	frameworks/native/data/etc/android.hardware.sensor.compass.xml:system/etc/permissions/android.hardware.sensor.compass.xml \
 	frameworks/native/data/etc/android.hardware.camera.front.xml:system/etc/permissions/android.hardware.camera.front.xml \
 	frameworks/native/data/etc/android.hardware.touchscreen.multitouch.jazzhand.xml:system/etc/permissions/android.hardware.touchscreen.multitouch.jazzhand.xml \
 	frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml \
-- 
1.8.0

